roles/ovirt-image-template/tasks/qcow2_image.yml

```diff
@@ -101,6 +108,12 @@
       - ovirt-template-image
 
   always:
+    - name: Remove downloaded image
+      file:
+        path: "{{ image_path }}"
+        state: absent
+      when:  image_cache_download == false
```

Comment:
Any reason to use `== false` instead of `not`

Reply:
@machacekondra Made the recommended changes